unit-testing/src/mock-function/data/connect-to-cosmos.ts

```diff
@@ -0,0 +1,41 @@
+// connect-to-cosmos.ts
+
+import { Container, CosmosClient } from '@azure/cosmos';
+import { DefaultAzureCredential } from '@azure/identity';
+import 'dotenv/config';
+import { v4 as uuidv4 } from 'uuid';
+
+export { Container };
+
+export function connectToCosmosWithKey() {
```

Comment:
Should this be part of the sample? We are moving towards managed identity and keyless authentication across the board so maybe our samples should make it easy to find the DefaultAzureCredential configuration and harder to find how to use key based authentication. Just a thought, no strong opinions here

Reply:
I removed it and added a Link in the readme to the Cosmos DB RBAC info.